src/theory/theory_inference_manager.cpp

```diff
@@ -84,6 +84,27 @@ void TheoryInferenceManager::trustedConflict(TrustNode tconf)
   }
 }
 
+void TheoryInferenceManager::conflictExp(PfRule id,
+                                         const std::vector<Node>& exp,
+                                         const std::vector<Node>& args)
+{
+  if (!d_theoryState.isInConflict())
```

Comment:
Do we actually want to simply drop a conflict? I'd guess we should at least communicate this fact (by a `bool` return value)...

Reply:
This is the same behavior as normal conflict/trustedConflict above.  I'm open to changing the return value later (maybe a separate PR?). I actually was tempted to change this to `Node` so that the theory is informed of what the conflict was (for debugging).